Classes/Core/Testbase.php

```diff
@@ -538,6 +538,7 @@ public function setUpBasicTypo3Bootstrap($instancePath)
             ->baseSetup()
             ->loadConfigurationAndInitialize(true)
             ->loadTypo3LoadedExtAndExtLocalconf(true)
+            ->initializeBackendRouter()
```

Comment:
I think it is better to call this in FunctionalTestCase.php at the two places directly before loadExtensionTables() is called.